workflow/commons/src/test/java/com/datastax/oss/dsbulk/workflow/commons/schema/QueryInspectorTest.java

```diff
@@ -633,6 +655,17 @@ static List<Arguments> should_detect_unsupported_selector() {
         arguments("SELECT * FROM ks.t1", false));
   }
 
+  @ParameterizedTest
+  @MethodSource
+  void should_detect_unsupported_vector_selector(String query, boolean expected) {
```

Comment:
Nit: do we really need a parameterized test here? Are you anticipating more test cases?

Reply:
Bah, good catch.  I think this is a holdover from when I was originally planning on some other tests which would involve various permutations of quoted values.  Obviously that doesn't apply at all here... I'll change it.